src/components/CustomerQuota/CheckoutUnsuccessful/CheckoutUnsuccessfulCard.tsx

```diff
@@ -91,13 +91,13 @@ export const CheckoutUnsuccessfulCard: FunctionComponent<CheckoutUnsuccessfulCar
         <View
           style={{
             ...sharedStyles.failureResultWrapper,
-            backgroundColor: theme.checkoutUnsuccessfulCard.backgroundColor,
+            backgroundColor: theme.checkoutUnsuccessCard.backgroundColor,
```

Comment:
oh ya, why does the wording becomes `checkoutUnsuccessCard` ya

Reply:
oh because i noticed it was named `UnsuccessCard` somewhere else. But checking again, noticed it's only at the storybook. Changed it back to `UnsuccessfulCard` [here](https://github.com/rationally-app/mobile-application/pull/499/commits/9d175ec6d124c8a2b9e6c1029e1c1a0403d0e892)!